### ISSUE TITLE
Manage and display talks assigned to events

### ISSUE THREAD

**Issue by nicholasjhenry** [montrealrb/Montreal.rb#60]

**Comment by VeraZab** [comment]
im working on this :)

**Comment by cawel** [comment]
@VeraZab Any progress?

The reason I'm asking is because it's somewhat related to another issue I'm working on: #83.

**Comment by VeraZab** [comment]
hello, we've submitted our work after the hack night...

**Comment by cawel** [comment]
The best would be to create a PR (pull-request) linking it back to that issue. If that sounds confusing, no worries. Let us know, and we'll find another way.

**Comment by VeraZab** [comment]
I think that we didn't completely finish this ticket, and so didn't submit. Sorry to be blocking you. Not sure how our work and this ticket is linked... Let me know what I can do to unblock..thanks

**Comment by cawel** [comment]
Not to worry @VeraZab. You are not blocking after all. No pressure :)

**Comment by karimmtarek** [comment]
I'm on it

**Comment by karimmtarek** [comment]
This bug #146 prevented me from creating new events from the admin UI, so I had to to remove `TBALocation.new` from the `location` method in the Event.rb, to be able to work with events and move on with the task at hand

**Comment by karimmtarek** [comment]
@nicholasjhenry according to the [domain model](https://github.com/montrealrb/Montreal.rb/wiki/Domain-Model) the association between members and talks are M:M? is that correct? if that's the case what is more preferable using HMABT to has_many through?
One more thing in the `talk_dashboard.rb` there is votes and author while in the domain model those are not mentioned can you please clarify, thanks :)

**Comment by nicholasjhenry** [comment]
> according to the domain model the association between members and talks are M:M? is that correct?

Let's keep it simple -- while a talk in very rare occasions maybe presented by two folks, we'll deal with that in the future. I've updated the domain model to show a talk is only presented by one member.

> One more thing in the talk_dashboard.rb there is votes and author while in the domain model those are not mentioned can you please clarify

The `talk_dashboard.rb` was generated using an old schema: https://github.com/montrealrb/Montreal.rb/issues/114

These need to be deleted, they should have never been merged into master -- so please ignore the ones listed in the issue.

**Comment by karimmtarek** [comment]
@nicholasjhenry thanks for the clarification :+1:

**Comment by karimmtarek** [comment]
@montrealrb/contributors Should the talk only be shown with the events or is it going to have it's own index page?
